src/rest_api.py

```diff
@@ -455,6 +455,40 @@ def get_stacks_report(report):
     return flask.jsonify(_s3_helper.get_object_content(report))
 
 
+@app.route('/api/v1/ingestion-report/list', methods=['GET'])
+def list_ingestion_reports():
+    """Endpoint to fetch the list of generated ingestion reports."""
+    return flask.jsonify(_s3_helper.list_objects("ingestion-data/epv"))
```

Comment:
@yzainee is this meant to be a hardcoded object name? Can we not make it an environment variable?

Reply:
this is the folder structure defined for the ingestion report. I didnt really think we need to have an env variable for that